canonical/vanilla-framework
# Fix dark theme dropdown option bug

## Done

Fixes [#4911](https://github.com/canonical/vanilla-framework/issues/4911), [WD-7399](https://warthogs.atlassian.net/browse/WD-7399)

## QA

- Use Windows/Ubuntu to recreate this bug. Mac will show the default OS dropdown stylings.
- Go to [dark forms](https://0.0.0.0:8108/docs/examples/patterns/forms/forms-dark) on local development
- Select an Ubuntu version from the dropdown
- Hover over other options.
- Selected option should be highlighted in grey, making the option readable.

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots
![Screen Recording 2023-11-22 at 2 56 19 PM](https://github.com/canonical/vanilla-framework/assets/62298176/eba2ccbc-69ae-4a5e-b574-56bfbe0722bf)




[WD-7399]: https://warthogs.atlassian.net/browse/WD-7399?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
